docs/source/api/user-guide/measurements/data.csv

```diff
@@ -0,0 +1,1000 @@
+59,1,9,1,0,1
```

Comment:
I think this is the PUMS data: we shouldn't need to add another copy to the repo.

(If there is some process that is copying and adding the CSV by mistake, that root problem should be fixed.)